### ISSUE TITLE
Incomplete suggestions for wordmap when Translation Identifier are in different cases

### ISSUE THREAD

**Issue by elsylambert** [unfoldingWord/translationCore#6575]
While testing for #6534, noticed that when importing a project with a different case of Translation Identifier, some of the suggestions do not show up.
Projects-
[hi_reg_luk_book.zip](https://github.com/unfoldingWord/translationCore/files/3876988/hi_reg_luk_book.zip) TI as- REG - not all suggestions show up
[hi_reg_eph_book.zip](https://github.com/unfoldingWord/translationCore/files/3876989/hi_reg_eph_book.zip) TI as reg- shows a lot more suggestions
Aligned verse 1 in this project 
[hi_reg_tit_book.zip](https://github.com/unfoldingWord/translationCore/files/3876998/hi_reg_tit_book.zip)
![image](https://user-images.githubusercontent.com/51806180/69381350-90a16700-0c82-11ea-9d10-50e504a0ca45.png)

The word "God " has same lemma but aligned differently, does this cause the wordmap suggestion to not show up?

**Comment by cckozie** [comment]
2.1.0 (5980326)
It looks to me like the noted problem is not related to the case of the translation ID. I'm seeing the same suggestions in a Luke project with a translation ID that matches that of the aligned book, and a Luke project with a case difference in the translation ID.

Upon further investigation I noticed that there was a slight difference in the Hindi word for God from the alignment in Titus and the ones I was looking at in Luke 1:6 and others. I did find a match for the Hindi word in Luke 20:37, and there the suggestion does show as expected.
<img width="1014" alt="luk-upper-REG-20-37" src="https://user-images.githubusercontent.com/8171759/69443789-bbd49680-0d14-11ea-869d-ca335292c056.png">
